Stripe/PublicHeaders/UI/STPPaymentCardTextField.h

```diff
@@ -220,6 +220,11 @@
  */
 @property(nonatomic, readonly, nullable) STPCardParams *card;
```

Comment:
I think it's fine to do that; that thing is deprecated anyway, and this would be an additive change.